src/server/routes/index.js

```diff
@@ -4,18 +4,14 @@ const users = require('../../models/db/users');
 
 const router = require('express').Router();
 
-router.get('/', (request, response) => {
-  response.render('todo/todo-list');
-});
-
 router.get('/signup', (request, response) => {
   response.render('users/sign-up');
 });
 
 router.post('/signup', (request, response) => {
   users.signUp(request.body)
     .then((user) => {
-      response.redirect('/');
+      response.redirect('/signin');
```

Comment:
might not get to it this week, but as a usability point it might be good to log someone in as soon as they're signed up. Unless there are some other security hoops they have to jump through.

Reply:
Gotcha! I did it this way since I was not sending a "verify" email or text message. But I get what you are saying. I will def look into this when I refactor ;)